resources/zib/ext-CarePlan-MaterialUsed.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="ext-CarePlan-MaterialUsed" />

```

Comment:
Should be at least ext-CarePlan.MaterialUsed I guess.

But why not ext-NursingIntervention.MedicalDevice? Although the current name seems logical if it is used more often within the CarePlan resource. Is it used more often?
Though the guidelines do not really offer a ext-[fhir resource type]-[description] choice.

Reply:
Because it is a generic extension to couple used materials to a CarePlan, not e specific extension for NursingIntervention concept MedicalDevice (profiling guidelines: "If that's not possible, try to create an extension in a reusable way (or reuse a previously defined extension)"
The name is given by this rule: "if the use context is a single resource:ext-[resource]-[purpose]".

One could argue that "MaterialUsed" could be "Device", but at the moment it is not known what the actual target will be and if there maybe will be multiple targets. IMO "MaterialUsed" is a descriptive enough name.

Reply:
Ok, missed that one